nuxt-frontend/components/services/CTASection.vue

```diff
@@ -1,50 +1,51 @@
 <template>
-  <section class="relative mt-32 xl:mt-40 mb-20 xl:mb-0">
-    <div class="xl:hidden">
+  <section
+    class="mt-16 md:mt-32 lg:mt-60 mb-20 lg:mb-0 lg:pb-28 lg:bg-gradient-to-t lg:from-orange-1 lg:via-pink-1 lg:to-white 3xl:outer-container"
+  >
+    <div class="lg:hidden relative mx-auto w-[400px]">
       <img
         :src="ctabgImages[0]"
         :srcset="`${ctabgImages[0]} 400w`"
         loading="lazy"
         alt="cta-bg-image"
-        class="absolute top-[-6.5rem] sm:left-[4.5rem] md:left-[6.5rem] w-[100%] sm:w-[80%] tw object-content"
+        class="absolute top-[-14] md:top-[-6.5rem] left-2 object-contain mx-auto"
       />
     </div>
-    <div class="container xl:flex xl:flex-row">
-      <div
-        class="flex flex-col items-center xl:items-start xl:mb-20 xl:w-[60%] 2xl:w-[50%]"
-      >
+    <div class="container flex flex-row">
+      <div class="flex flex-col items-center lg:items-start lg:w-3/5">
         <p
-          class="font-inter-bold text-center xl:text-left text-[1.875rem] md:text-[2.813rem] lg:text-[3.438rem] leading-9 md:leading-[3.75rem] lg:leading-[5.156rem] font-inter-bold text-black-core/[0.87]"
+          class="text-center lg:text-left mobile-header-2 lg:desk-header-2 text-black-87 w-[69%] lg:w-[87%] 2xl:w-[82%]"
```

Comment:
Maybe you can provide full width for mobile view

Reply:
No its given intensionally to set image image and text